Mixmatch/streetview_dataset/parse_TFRecord.ipynb

```diff
@@ -0,0 +1,221 @@
+    "    img.save(imgByteArr, format='JPEG')\n",
+    "    imgByteArr = imgByteArr.getvalue()\n",
+    "\n",
+    "    example = tf.train.Example(features=tf.train.Features(feature={\n",
+    "        \"image\": tf.train.Feature(bytes_list=tf.train.BytesList(value=[imgByteArr])),\n",
+    "        \"label\": tf.train.Feature(int64_list=tf.train.Int64List(value=[label]))}))\n",
+    "    \n",
+    "    return example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def write_TFRecord_UCF(parsed_image_dataset, folder_path, writer):\n",
+    "    for image_features in parsed_image_dataset:\n",
+    "        img_name, confidence, bbox = parse_labels(image_features)\n",
+    "\n",
+    "        if img_name:\n",
+    "            label = 0\n",
+    "            if confidence.size>0 and max(confidence)>0.9:\n",
+    "                label = 1\n",
+    "                pos = np.argmax(confidence)\n",
+    "                bbox = bbox[:, pos]\n",
+    "            elif confidence.size==0 or (confidence.size>0 and max(confidence)<0.1):\n",
+    "                label = 0\n",
+    "            else:\n",
+    "                continue\n",
+    "\n",
+    "            img_path = os.path.join(folder_path, img_name)\n",
+    "            try:\n",
+    "                img = Image.open(img_path, \"r\")\n",
+    "\n",
+    "                # Exclude all non RGB images\n",
+    "                if len(img.getbands()) != 3:\n",
+    "                    continue\n",
+    "\n",
+    "                if label:\n",
+    "                    img = img.crop(bbox)\n",
+    "                img = img.resize((64, 64))\n",
+    "\n",
+    "                example = img2example(img, label)\n",
+    "                writer.write(example.SerializeToString())\n",
+    "            except Exception as e:\n",
+    "                print(e)\n",
+    "                print(img_path + \" is not valid\")\n",
+    "\n",
+    "def write_TFRecord_TC11(folder_path, writer):\n",
+    "    for img_name in os.listdir(folder_path):\n",
+    "        label = 1\n",
+    "        img_path = os.path.join(folder_path, img_name)\n",
+    "        \n",
+    "        try:\n",
+    "            img = Image.open(img_path, \"r\")\n",
+    "\n",
+    "            # Exclude all non RGB images\n",
+    "            if len(img.getbands()) != 3:\n",
+    "                continue\n",
+    "            \n",
+    "            img = img.resize((64, 64))\n",
+    "\n",
+    "            example = img2example(img, label)\n",
+    "            writer.write(example.SerializeToString())\n",
+    "        except Exception as e:\n",
+    "            print(e)\n",
+    "            print(img_path + \" is not valid\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# input tfrecord file path\n",
+    "input_record_path = './streetlearn-detections/'\n",
```

Comment:
Use CAPS_WITH_UNDER for global constants: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#3164-guidelines-derived-from-guidos-recommendations

Reply:
Done